### ISSUE TITLE
Move or copy contents of RELEASE-NOTES.md to the release pages

### ISSUE THREAD

**Issue by waldyrious** [mwclient/mwclient#109]
Here: https://github.com/mwclient/mwclient/releases. This would prevent duplication.

One relevant question is: would that info stop being tracked in the git repo? Or are the release descriptions kept as (long-form) commit messages in the tags? Even so, would it be preferred to have a text file that could be read in a local clone rather than running a git command to list the tags, or are we ok with providing a link to the github release pages?

@danmichaelo what do you think?

**Comment by danmichaelo** [comment]
I've been thinking about this as well. I don't think the release notes from the Github release pages are available outside Github, and that's been my reason for keeping `RELEASE-NOTES.md` so far, but it's a chore having to update it, and we end up with a lot of "Update release notes" commits, so I would be happy to get rid of it. A link to the github release pages should probably suffice. While Github will go away some day, I don't expect it to happen any time soon.

**Comment by waldyrious** [comment]
Agreed. Current documentation makes sense to live with the repo, but history of changes isn't that critical. I'll take the time to move the notes into the releases page once I get the chance, then.

**Comment by danmichaelo** [comment]
Great!

**Comment by danmichaelo** [comment]
Didn't bother copying everything into RELEASE-NOTES-md yet in case we just remove the file:
https://github.com/mwclient/mwclient/commit/48b7d05f190ac8e0ca7f58497ee17166ce7b4042

**Comment by waldyrious** [comment]
I understand that point of view, and agree that those points are valid concerns. No objection to retain RELEASE-NOTES.md as the primary source of this information, and only copy them to the Github release pages for convenience.

Unfortunately, I can't promise I'll be able to review this and do whatever needs to be done anytime soon, but a checklist of the tasks to perform, according to the points agreed here and on #178, would already be helpful :)

**Comment by AdamWill** [comment]
Well, I was really just registering an opinion - I understand if @danmichaelo doesn't want to go that way, because however you work it it's likely to involve more manual work with each release.

**Comment by danmichaelo** [comment]
I'm ok with keeping the file, but I won't promise that I always remember to update it. We can aim for eventual consistency :)
